Prog/Voices_q15_recode.sas

```diff
@@ -801,3 +805,70 @@ run;
 
 filename fexport clear;
 
+*ADD CODE TO GET # of PEOPLE WHO SAID X*;
+
+proc summary data=voices_Q15_recode;
+	by caseid;
+	var Q15_3: ;
+	output out=voices_Q15_recode_sum_0 sum=;
+run; 
+
+data voices_Q15_recode_sum (drop=Q15: _type_ _freq_);
+	
+	Merge voices_Q15_recode_sum_0
+		  Voices.VoicesDMVSurvey2017 (keep=caseid weight dov_urban ppethm ppracem ppeducat ppincimp ppage ppgender pprent  PPT01 PPT25 PPT612 PPT1317);
+	by caseid;
+
+	%make_break_vars_2017;
+
+	sQ15_3a=.;
```

Comment:
Nothing wrong with this but you don't need to initialize new vars to missing. They will be set to missing by default.